api/scpca_portal/management/commands/load_data.py

```diff
@@ -79,12 +74,28 @@ def add_arguments(self, parser):
             "--update-s3", action=BooleanOptionalAction, default=settings.UPDATE_S3_DATA
         )
 
-    def clean_up_input_data(self, project):
-        shutil.rmtree(common.INPUT_DATA_PATH / project.scpca_id, ignore_errors=True)
-
     def handle(self, *args, **kwargs):
         self.load_data(**kwargs)
 
+    @staticmethod
+    def project_has_s3_files(project_id: str) -> bool:
+        return any(
+            True
+            for project_path in common.INPUT_DATA_PATH.iterdir()
+            if project_path.name == project_id and project_path.is_dir()
+            for nested_path in project_path.iterdir()
+            if nested_path.is_dir()
+        )
+
+    @staticmethod
+    def clean_up_input_data(project):
+        shutil.rmtree(common.INPUT_DATA_PATH / project.scpca_id, ignore_errors=True)
+
+    @staticmethod
+    def clean_up_output_data():
+        for path in Path(common.OUTPUT_DATA_PATH).glob("*"):
+            path.unlink(missing_ok=True)
+
```

Comment:
This commit `770d404` is actually supposed to be in the branch from PR https://github.com/AlexsLemonade/scpca-portal/pull/836. I'm not sure how it was added to this branch, I feel like it must have had something to do with the commit being shared during a rebase (as this branch was used as the base for the branch in PR https://github.com/AlexsLemonade/scpca-portal/pull/836 branch).

Reply:
I think we can avoid this by using merge by default from now on. Don't cherry pick changes into your branch unless needed to resolve something. Rebasing can be done until you push remote changes. But honestly, 99% of the time we just want to merge changes up and keep git history as is.